vxtelegram/tests/test_telegram.py

```diff
@@ -534,18 +534,45 @@ def test_outbound_query_reply_no_errors(self):
 
         # Ignore statuses published on transport startup
         [_, __, status] = yield self.helper.wait_for_dispatched_statuses()
-        self.assert_status(status, {
+        self.assert_dict(status, {
             'status': 'ok',
             'component': 'telegram_outbound',
             'type': 'good_outbound_request',
             'message': 'Outbound request successful',
         })
 
     @inlineCallbacks
+    def test_outbound_query_reply_unexpected_format(self):
+        """
+        We should log and publish a nack when a query reply does not contain a
+        results field. We don't publish a down status, since this
+        event has no bearing on the transport's health.
```

Comment:
I think we should actually publish a `down` status. Yes, the transport probably hasn't done anything wrong, but people's messages aren't being delivered and they probably want to know that.